docs/table-optimization.md

```diff
@@ -0,0 +1,33 @@
+# Fluo Table Optimization
+
+## Background
+
+Recipes may need to make Accumulo specific table modification for optimal
+performance.  Configuring the Accumulo tablet balancer and adding splits are
+two optimizations that are currently done.  Offering a standard way to do these
+optimizations makes it easier to use recipes correctly.  These optimizations
+are optional.  You could skip them for integration testing, but would probably
+want to use them in production.
+
+## Example
+
+```java
+
+FluoConfiguration fluoConf = ...
+
+//Post initialization table optimizations
+Pirtos pirtos = new Pirtos();
```

Comment:
ok, let me think about this some more.  Getting rid of that class would be nice.